be/extmap.c

```diff
@@ -671,9 +755,13 @@ M0_INTERNAL void m0_be_emap_obj_insert(struct m0_be_emap *map,
 	m0_format_footer_update(&map->em_key);
 	map->em_rec.er_start = 0;
 	map->em_rec.er_value = val;
+	map->em_rec.er_cksum_nob = 0;
+	emap_rec_init(&map->em_rec);
 	m0_format_footer_update(&map->em_rec);
```

Comment:
emap_rec_init does m0_format_footer_update(), so lets remove line 760 for optimization